docker-compose.yml

```diff
@@ -0,0 +1,183 @@
+version: '3.4'
+
+volumes:
+  fcrepo:
+  solr:
+  db:
+  redis:
+  zk1:
+  zkconf:
+  app:
+  assets:
+  cache:
+
+networks:
+  internal:
+
+services:
+  app: &app
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: DEVELOPMENT_IMAGE
+    image: hyku-api/development_image:latest
+    environment: &web_environment
+      HOST: "0.0.0.0" # Docker cannot forward to a containers localhost, so we need to use this IP instead
```

Comment:
Can you check if any of these settings can be removed as they're not required for the API?